packages/@react-spectrum/dialog/chromatic/intl/ar-AE.json

```diff
@@ -0,0 +1,25 @@
+{
+  "koji": "كوجي",
+  "kojiFoods": "الأطعمة كوجي",
+  "foodsMakeWithKoji": "الأطعمة اليابانية والتوابل والمشروبات المصنوعة من الكوجي",
+  "kojiColors": "ألوان كوجي",
+  "white": "أبيض (أخضر)",
+  "yellow": "أصفر",
+  "black": "أسود",
+  "name": "اسم",
+  "description": "وصف",
+  "soySauce": "شويو",
+  "soySauceDescription": "بهار سائل مصنوع من فول الصويا وكوجي والقمح.",
+  "miso": "ميسو",
+  "misoDescription": "توابل يابانية تقليدية.",
+  "amazake": "أمازاكي",
+  "amazakeDescription": "مشروب أرز حلو.",
+  "sake": "من أجل",
+  "sakeDescription": "نبيذ الأرز.",
+  "mirin": "ميرين",
+  "mirinDescription": "نبيذ أرز حلو.",
+  "riceWineVinegar": "خل نبيذ الأرز",
+  "riceWineVinegarDescription": "خل مصنوع من نبيذ الأرز.",
+  "cancel": "يلغي",
+  "confirm": "Confيتأكدirm"
```

Comment:
left a few comments in chromatic, but i think some of these strings are wrong, just a copy paste error it looks like